lib/user.rb

```diff
@@ -0,0 +1,14 @@
+require 'data_mapper'
+
+class User
+
+	include DataMapper::Resource
+
+	#attr_accessor :password
```

Comment:
Incorrect indentation detected (column 1 instead of 2).<br>Missing space after #.<br>Tab detected.